### PR TITLE
2022 q4 hof

### PR DESCRIPTION
**Pull request by tomrittervg**
@fkiriakos07

### CODE REVIEW

File: bug-bounty-hof/web.yml [hunk 1]
```diff
@@ -68,8 +101,7 @@ names:
   date: 2022-05-16
 - name: Ali Syarief
   date: 2022-05-11
-  twitter: "@kang_ali"
-  url: https://twitter.com/@kang_ali
```
Comment (fkiriakos07) [inline bug-bounty-hof/web.yml:72]: did Kang change the way they are mentioned?

File: bug-bounty-hof/web.yml [hunk 2]
```diff
@@ -1,4 +1,37 @@
 names:
+- name: Jake Nicholson
+  date: 2022-12-19
+  url: https://www.linkedin.com/in/dimedaughter/
+- name: redparsec
+  date: 2022-11-28
+  twitter: "@r3dpars3c"
+  url: https://twitter.com/@r3dpars3c
+- name: Leonardo Junior
+  date: 2022-11-25
```
Comment (fkiriakos07) [inline bug-bounty-hof/web.yml:10]:
```suggestion
  date: 2022-11-25
  url: https://github.com/Leommjr
```

File: bug-bounty-hof/web.yml [hunk 3]
```diff
@@ -68,8 +101,7 @@ names:
   date: 2022-05-16
 - name: Ali Syarief
   date: 2022-05-11
-  twitter: "@kang_ali"
-  url: https://twitter.com/@kang_ali
+  url: https://www.linkedin.com/in/mohammad-ali-syarief/
 - name: Brian Carpenter of Geeknik Labs & Farm
```
Comment (fkiriakos07) [inline bug-bounty-hof/web.yml:105]: @tomrittervg this reporter declined to be mentioned in HoF https://bugzilla.mozilla.org/show_bug.cgi?id=1796601#c10

File: bug-bounty-hof/web.yml [hunk 4]
```diff
@@ -1,4 +1,37 @@
...
+  url: https://danielv.com.br/
+- name: Griffin Francis
+  date: 2022-10-31
+- name: Muhammad Zaid Ghifari
+  date: 2022-10-24
+- name: Manojkumar J
+  date: 2022-10-05
+  url: https://www.linkedin.com/in/manojkumar-j-7ba35b202/
 - name: David Klein
   date: 2022-09-21
```
Comment (fkiriakos07) [inline bug-bounty-hof/web.yml:36]:
```suggestion
  date: 2022-09-21
  twitter: "@ncd_leen"
  url: https://twitter.com/ncd_leen
```

**Review by fkiriakos07 — CHANGES_REQUESTED** [review]

**Comment by fkiriakos07** [comment]
Hello @tomrittervg , apologies for the delay, thank you for generating the HoF, I suggested some changes based on the reporters' requests, please check.

**Review by dveditz — APPROVED** [review]
lgtm, but I don't know which concerns Frida wanted addressed

File: bug-bounty-hof/web.yml (hunk 3 above)
Comment (tomrittervg) [reply bug-bounty-hof/web.yml:105]: He's not in Q4; this entry is for some other bug in Q2..

**Comment by tomrittervg** [comment]
@fkiriakos07 Can you double check this?

File: bug-bounty-hof/web.yml (hunk 1 above)
Comment (fkiriakos07) [reply bug-bounty-hof/web.yml:72]: got it

**Review by fkiriakos07 — APPROVED** [review]

**Comment by fkiriakos07** [comment]
thanks @tomrittervg looks good to me, one question, do you know how this reporter is mentioned? https://bugzilla.mozilla.org/show_bug.cgi?id=1783267 I couldn't find any mention using his alias or name

**Pull request merged**